monai/data/image_reader.py

```diff
@@ -219,6 +220,16 @@ def _get_meta_dict(self, img) -> Dict:
             # ignore deprecated, legacy members that cause issues
             if key.startswith("ITK_original_"):
                 continue
+            if (
+                key == "NRRD_measurement frame"
+                and int(itk.Version.GetITKMajorVersion()) == 5
+                and int(itk.Version.GetITKMinorVersion()) < 2
```

Comment:
This wouldn't work for ITK v4. Would need to be more like:

```python
if key == "NRRD" and (vMaj < 5 or vMin < 2)
```

Reply:
I think `((vMaj < 5) or (vMaj == 5 or vMin < 2))` right?

But does MONAI support ITK < 5? Should this section change too?
https://github.com/Project-MONAI/MONAI/blob/master/monai/data/image_reader.py#L128-L132

Reply:
perhaps we should have a min. verison at the project level, `itk>=5.0`? @Nic-Ma @hjmjohnson we don't have thorough tests for all the versions, but it seems the demos are built with itk 5.x

Reply:
I think @aylward and @thewtex would like the minimum version to be set to `itk==5.2` once that's released. In the meantime, setting it to `itk==5.0` is fine.